scripts/toaster_configuration

```diff
@@ -189,8 +189,18 @@ EOF
 }
 
 create_toaster_venv () {
-    python -m virtualenv -p python3 "$BUILDDIR/venv"
-    python -m virtualenv -p python2 "$BUILDDIR/venv"
+    if [ $(lsb_release -i | awk {'print $3'}) == "RedHatEnterpriseWorkstation" ]; then
+        # If its RHEL 7 there is a bug in virtaulenv which has to be tweaked
+        if [ $(lsb_release -r | awk {'print $2'} | cut -d "." -f1) -eq 7 ]; then
+            easy_install --user --upgrade virtualenv
+            export PATH=$HOME/.local/bin:$PATH
+            python -m virtualenv -p python3 "$BUILDDIR/venv"
+            python -m virtualenv -p python2 "$BUILDDIR/venv"
+        fi
+    else
+        python -m virtualenv -p python3 "$BUILDDIR/venv"
+        python -m virtualenv -p python2 "$BUILDDIR/venv"
```

Comment:
The python -m lines are duplicated. Just do the easy_install & PATH adjustment in the rhel7 block and then run the python lines outside the conditional in all cases.